src/bridge/cockpitschedsamples.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "config.h"
+#include "cockpitschedsamples.h"
+
+#include <stdlib.h>
+
+void
+cockpit_sched_samples (CockpitSamples *samples)
+{
+  const char * const instances[] = { "1min", "5min", "15min" };
+  double loadavg[3];
+  int count;
+
+  count = getloadavg(loadavg, 3);
+  if (count < 0) {
+      g_message ("getloadavg() failed: %m");
```

Comment:
If this error happens at all, I would expect it to happen on every sample, which would spam the journal.  Maybe logging it once is enough?

Reply:
Good point! Done.